pkg/workloadmeta/store.go

```diff
@@ -396,29 +440,29 @@ func (s *store) handleEvents(evs []CollectorEvent) {
 			entitiesOfKind = s.store[meta.Kind]
 		}
 
-		entityOfSource, ok := entitiesOfKind[meta.ID]
+		entitiesBySource, ok := entitiesOfKind[meta.ID]
```

Comment:
This is a good rename, but this has also been changed in #10331. It would save us both a lot of work to just revert and avoid the conflicts.

Reply:
I'll drop that commit when I rebase onto #10331.